aladdin/commands/deploy.py

```diff
@@ -120,8 +100,16 @@ def deploy(
         }
         # Update with cluster rule values
         values.update(cr.values)
+        # Update with user-specified values file
+        values_files = [
+            f"--values={file_name}" for file_name in set_override_values
+            if "=" not in file_name
+        ]
         # Update with --set-override-values
-        value_overrides = {k: v for k, v in (value.split("=") for value in set_override_values)}
+        value_overrides = {
+            k: v for k, v in
+            (value.split("=") for value in set_override_values if "=" in value)
+        }
```

Comment:
i'm always a fan of comprehensions, but maybe this is better expressed as

```
value_overrides = dict(value.split("=") for value in set_override_values if "=" in value)
```

Reply:
done!